partition/roles/metal-core/files/bgp-neighbor-stats.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e 
+tmpfile=$(mktemp /var/run/bgp-neighbors/bgp-neighbors.XXXXXX)
+destfile=/var/run/bgp-neighbors/bgp-neighbors.json
+
+/usr/bin/vtysh -c "show ip bgp vrf all neighbors json" > "${tmpfile}"
+
+mv "${tmpfile}" "${destfile}"
+rm "${tmpfile}"
```

Comment:
If the script fails for any reason, `metal-core` still believes there are BGP neighbors. Please remove the destination file at the beginning of the script.